cfpb/sbl-frontend
# feat: buildspec for sbl-frontend image build

Executed by a Codebuild project, this buildspec builds a docker image off the preview branch, push the image to ECR, and restarts the pod associated with the sbl-frontend deployment.

## Changes

- Created a `buildspec.yml` with relevant commands.

## How to test this PR

1. Will need to create a preview branch, and work with the developer's to ensure the image is being built properly when a change occurs to the preview branch.
